proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -33,6 +33,14 @@ service Query {
   rpc ModuleAccountByName(QueryModuleAccountByNameRequest) returns (QueryModuleAccountByNameResponse) {
     option (google.api.http).get = "/cosmos/auth/v1beta1/module_accounts/{name}";
   }
+
+  // NextAccountNumber queries the global account number.
+  // Please be careful use this rpc. This rpc can be disappear whenever.
+  // And backward compatibility is not guaranteed.
+  rpc NextAccountNumber(QueryNextAccountNumberRequest) returns (QueryNextAccountNumberResponse) {
+    option deprecated            = true;
+    option (google.api.http).get = "/cosmos/auth/v1beta1/next_account_index";
```

Comment:
It's using `index`, while the others uses `number`. Is it on purpose?

Reply:
Oh, I mistook it. I'll fix it.